src/pages-and-resources/data/selectors.js

```diff
@@ -2,3 +2,4 @@
 
 export const getLoadingStatus = (status) => status.pagesAndResources.loadingStatus;
 export const getSavingStatus = (status) => status.pagesAndResources.savingStatus;
+export const getApiCallStatus = (status) => status.pagesAndResources.ApiCallStatus;
```

Comment:
there could be multiple API calls within this MFE, how this is distinguishing with API call status this is returning?